src/PotentialEnergyEquationTerms.jl

```diff
@@ -18,16 +31,48 @@ Return a `KernelFunctionOperation` to compute the `PotentialEnergy` per unit vol
 ```math
 Eₚ = \\frac{gρz}{ρ₀}
 ```
-at each grid `location` in `model`.
+at each grid `location` in `model`. `PotentialEnergy` is defined for both `BuoyancyTracer`
+and `SeawaterBuoyancy`. See the relevant Oceananigans.jl documentation on
+[buoyancy models](https://clima.github.io/OceananigansDocumentation/dev/model_setup/buoyancy_and_equation_of_state/)
+for more information about available options.
 
-**NOTE:** A `BoussinesqEquationOfState` must be used in the `model` to calculate
-`seawater_density`. See the [relevant documentation](https://clima.github.io/OceananigansDocumentation/dev/model_setup/buoyancy_and_equation_of_state/#Idealized-nonlinear-equations-of-state)
-for how to set `SeawaterBuoyancy` using a `BoussinesqEquationOfState`.
+The optional keyword argument `geopotential_height` is only used
+if ones wishes to calculate `Eₚ` with a potential density referenced to `geopotential_height`,
+rather than in-situ density, when using a `BoussinesqEquationOfState`.
 
 Example
 =======
 
-The default behaviour of `PotentialEnergy` uses the *in-situ density* in the calculation:
+Usage with a `BuoyancyTracer` buoyacny model
+```jldoctest
+julia> using Oceananigans
+
+julia> using Oceanostics.PotentialEnergyEquationTerms: PotentialEnergy
+
+julia> grid = RectilinearGrid(size=100, z=(-1000, 0), topology=(Flat, Flat, Bounded))
+1×1×100 RectilinearGrid{Float64, Flat, Flat, Bounded} on CPU with 0×0×3 halo
+├── Flat x
+├── Flat y
+└── Bounded  z ∈ [-1000.0, 0.0]   regularly spaced with Δz=10.0
+
+julia> model = NonhydrostaticModel(; grid, buoyancy=BuoyancyTracer(), tracers=(:b,))
+NonhydrostaticModel{CPU, RectilinearGrid}(time = 0 seconds, iteration = 0)
+├── grid: 1×1×100 RectilinearGrid{Float64, Flat, Flat, Bounded} on CPU with 0×0×3 halo
+├── timestepper: QuasiAdamsBashforth2TimeStepper
+├── tracers: b
+├── closure: Nothing
+├── buoyancy: BuoyancyTracer with ĝ = NegativeZDirection()
+└── coriolis: Nothing
+
+julia> PotentialEnergy(model)
```

Comment:
This should be `potential_energy` not `PotentialEnergy`...

Reply:
Yes I agree. As mentioned [here](https://github.com/tomchor/Oceanostics.jl/pull/164#issuecomment-2019231307) I think it would be best to change the whole package but I guess could start here.